editoast/editoast_schemas/src/infra/railjson.rs

```diff
@@ -17,6 +17,10 @@ use super::TrackSection;
 
 pub const RAILJSON_VERSION: &str = "3.4.13";
 
+pub fn min_version_prefix() -> String {
```

Comment:
Is it "minimum" version accepted or the "minor" from semantic versioning? I would either expand to `minimum_version` or `major_version` (or `railjson_major_version`).